config/webpack.watch.config.js

```diff
@@ -5,12 +5,8 @@ const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin');
 const baseWebpackConfig = require('./webpack.base.config');
 const project = require('./project');
 
-const GLOBALS = {
-  'process.env': { 'NODE_ENV': JSON.stringify('development') },
```

Comment:
you're passing in `NODE_ENV` with `npm run start` but this no longer exists. Does setting `mode: 'development'` in config handle this for you?

Reply:
Yes webpack 4 detects it automatically, but it is not because of `mode` it uses DefinePlugin under the hood.

Reply:
nice! we should update the plugin template with this too!!

Reply:
this all was done for the plugin template first: https://github.com/mongodb-js/compass-plugin/pull/23